js/README.md

```diff
@@ -1,3 +1,34 @@
-## Atlas JS
+# Atlas JS
 
-TBD.
+Welcome to TypeScript/JavaScript code backing the Atlas Design system. While Atlas endeavors to find CSS-only strategies for creating modern UI patterns, there are times when writing behavior code is necessary. These cases tend to fall into a one of several categories.
+
+- A component's markup / attributes require changing upon interaction for compatibilty with screenreaders or other assistive technology.
```

Comment:
thinking the folders should be something like this
```
behaviors/           // behaviors added by decorating an element with classes or data attributes
  dismiss.ts
  popover.ts
  show-more.ts
elements/           // web components
  local-time.ts
  form-validation.ts

Reply:
I like it

Reply:
updated